### DISCUSSION TITLE
How to create documentation for API built Gridify?

### DISCUSSION THREAD

**Discussion by moxplod** [alirezanet/Gridify#215] [General]
Hi, I plan to build a web API with Gridify. I am building a public facing API and it's documentation from scratch after a while.
Curious to hear what others have done here and the best way to set up the OpenAI documentation for Gridify inputs. 

Specifically for something like this https://alirezanet.github.io/Gridify/example/api-controller

```c#

public Paging<User> GetUsers([FromQuery] GridifyQuery query)
{
    return context.Users.Gridify(query);
}

``` 

We would need to provide our users how to filter, sort, page etc in the API documentation.

**Comment by erwinkramer** [comment]
Please check out https://github.com/alirezanet/Gridify/discussions/232. Probably has everything you need and more!

**Reply by moxplod** [reply]
Great - will take a look.
